axelor-mobile/src/main/java/com/axelor/apps/mobile/web/AppMobileController.java

```diff
@@ -37,12 +37,29 @@ public void getAppMobile(ActionRequest request, ActionResponse response) {
     AppMobile appMobile = (AppMobile) Beans.get(AppService.class).getApp("mobile");
 
     data.put("isAppMobileEnable", appMobile.getActive());
-    data.put("isSaleAppEnable", appMobile.getIsSaleAppEnable());
-    data.put("isCrmAppEnable", appMobile.getIsCrmAppEnable());
-    data.put("isTimesheetAppEnable", appMobile.getIsTimesheetAppEnable());
-    data.put("isLeaveAppEnable", appMobile.getIsLeaveAppEnable());
-    data.put("isExpenseAppEnable", appMobile.getIsExpenseAppEnable());
-    data.put("isTaskAppEnable", appMobile.getIsTaskAppEnable());
+    data.put(
+        "isSaleAppEnable",
+        Beans.get(AppService.class).isApp("sale") ? appMobile.getIsSaleAppEnable() : false);
```

Comment:
better inject app service instead of creating instance several time with 'Beans'